packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/ForceFunctionContainerStartExecutor.ts

```diff
@@ -1,148 +0,0 @@
-/*
- * Copyright (c) 2022, salesforce.com, inc.
- * All rights reserved.
- * Licensed under the BSD 3-Clause license.
- * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-
-import { Benny, getFunctionsBinary } from '@heroku/functions-core';
-import * as vscode from 'vscode';
-import { channelService } from '../../../channels';
-import { nls } from '../../../messages';
-import { notificationService } from '../../../notifications';
-import { telemetryService } from '../../../telemetry';
-import { FunctionService } from '../functionService';
-import { FUNCTION_RUNTIME_DETECTION_PATTERN } from '../types/constants';
```

Comment:
`FUNCTION_RUNTIME_DETECTION_PATTERN` is now unused:
https://github.com/forcedotcom/salesforcedx-vscode/blob/7a5f09583630a1518787abf8b28ab95d34460cff/packages/salesforcedx-vscode-core/src/commands/functions/types/constants.ts#L30-L35